### PR TITLE
Refactor portfolio

### PR DESCRIPTION
**Pull request by rica213**
Hi, 

In this PR, here is the list of things I did:
- Changed the structure of the portfolio section
- Added real projects in the portfolio section
- Updated the social media links
- Added functionality to download my resume
- Move all images in the assets/images directory
- Used font awesome for icons
- Fix inconsistent styling
- Updated skills section
- Added USP in my introduction
- Added a CTA button in the headline section
- Set navigation within project details as a future feature

Kindly review those changes. Thank you 💛

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,129 +41,59 @@
...
+            Larissa
           </h1>
           <p class="section__desc">
-            Hello! I am a software developer! I can help you build a product,
+            Hello! I am a Full-Stack Web Developer! <span class="usp">I translate vision into code.</span> I can help you build a product,
             feature or website. Take a look of my works. If you like what you
-            see and have a project you need coded, don’t hesitate and contact
+            see and have a project you need coded, don&apos;t hesitate and contact
             me.
           </p>
```
Comment (shaaibu7) [inline index.html:52]: This text has some grammatical errors, Kindly fix it. you can use Grammarly to correct this.

**Review by shaaibu7 — CHANGES_REQUESTED** [review]
Hi @rica213 

You did a great job working on your portfolio but you need to make some improvements before you move forward.



This text has some grammatical errors. you can use Grammarly to make this correction  :point_down: 
Screenshot from 2023-07-31 20-33-36

The spelling below should be Capybara :point_down: 

Screenshot from 2023-07-31 20-34-10

Your header should be fixed at the top for both mobile and desktop versions, kindly implement that project requirement.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shaaibu7 — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @rica213     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rica213** [comment]
Thank you time Shaaibu ❤️ @shaaibu7

**Pull request merged**